### PR TITLE
feat(x/gov): remove ability for validators to add votes

### PR DESCRIPTION
**Pull request by giunatale**
## Description

Closes: #7

---

### Author Checklist

*All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues.*

I have...

* [x] Included the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
* [ ] Added `!` to the type prefix if API, client, or state breaking change (i.e., requires minor or major version bump)
* [x] Targeted the correct branch (see [PR Targeting](https://github.com/atomone-hub/govgen/blob/main/CONTRIBUTING.md#pr-targeting))
* [x] Provided a link to the relevant issue or specification
* [x] Followed the guidelines for [building SDK modules](https://github.com/cosmos/cosmos-sdk/blob/main/docs/docs/building-modules)
* [x] Included the necessary unit and integration [tests](https://github.com/atomone-hub/govgen/blob/main/CONTRIBUTING.md#testing)
* [x] Added a changelog entry in `.changelog` (for details, see [contributing guidelines](../../CONTRIBUTING.md#changelog))
* [ ] Included comments for [documenting Go code](https://blog.golang.org/godoc)
* [ ] Updated the relevant documentation or specification
* [ ] Reviewed "Files changed" and left comments if necessary 
* [x] Confirmed all CI checks have passed

### Reviewers Checklist

*All items are required. Please add a note if the item is not applicable and please add
your handle next to the items reviewed if you only reviewed selected items.*

I have...

* [ ] confirmed the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
* [ ] confirmed `!` in the type prefix if API or client breaking change
* [ ] confirmed all author checklist items have been addressed 
* [ ] reviewed state machine logic
* [ ] reviewed API design and naming
* [ ] reviewed documentation is accurate
* [ ] reviewed tests and test coverage

### CODE REVIEW

**Comment by tbruyelle** [comment]
One question before merging, don't you think this should only apply to text proposals? I'm afraid we'll have trouble getting a quorum for param changes and upgrade proposals if validators can't vote with their delegation power.

File: CHANGELOG.md [hunk 1]
```diff
@@ -12,5 +12,7 @@
 
 ### FEATURES
 
-### STATE BREAKING
+* Forked `x/gov` module from Cosmos SDK v0.46.16 for custom moficiations.
+* Removed ability for validators to vote on proposals.
```
Comment (tbruyelle) [inline CHANGELOG.md:16]: We should add a link to the related PR

Comment (tbruyelle) [reply CHANGELOG.md]: done

**Comment by giunatale** [comment]
Let's merge for now. I understand what you are saying though. Maybe worth tracking this with an issue!

**Pull request merged**